packages/webapp/src/containers/Animals/Inventory/useAnimalsExist.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import { useGetAnimalsQuery, useGetAnimalBatchesQuery } from '../../../store/api/apiSlice';
+import useQueries from '../../../hooks/api/useQueries';
+
+const useAnimalsExist = () => {
+  const { data, isLoading } = useQueries([
+    { label: 'animals', hook: useGetAnimalsQuery },
+    { label: 'animalBatches', hook: useGetAnimalBatchesQuery },
+  ]);
+
+  const animalsExistOnFarm = !isLoading && (data.animals.length || data.animalBatches.length);
```

Comment:
This includes removed animals, so it should be:
```
[...data.animals, ...data.animalBatches].filter((entity) => !entity.animal_removal_reason_id).length;
```

Reply:
Will implement this in my PR

Reply:
I wonder why I thought we filtered them out! Thanks for your attention to detail!